src/components/HeaderWithBackButton/headerWithBackButtonPropTypes.js

```diff
@@ -96,6 +96,9 @@ const propTypes = {
 
     /** Whether we should navigate to report page when the route have a topMostReport  */
     shouldNavigateToTopMostReport: PropTypes.bool,
+
+    /** Whether we should overlay the 3 dots menu */
+    shouldOverlayDots: PropTypes.bool,
```

Comment:
I am not sure about the introduction of this new prop.
Can you share context of what issue you faced without this prop?

Reply:
Basically I wanted to add this style which overlays the entire header, but there was already a prop called `shouldOverlay`. In reality, that prop only actually affected the dots not the header, so I just made it more specific https://github.com/Expensify/App/pull/32154/files#diff-253571ee29dc35d72a8a0d7b0bf16002a7babd36a4e2ea60c8f02afa47e09a1cR70